src/UbuntuToolkit/sortfiltermodel.cpp

```diff
@@ -52,40 +51,43 @@ UT_NAMESPACE_BEGIN
  *         }
  *         ListElement {
  *             title: "Elephants Dream"
- *             // lowercase b
- *             producer: "blender"
+ *             producer: "Blender"
  *         }
  *         ListElement {
  *             title: "Big Buck Bunny"
- *             producer: "Blender"
+ *             producer: "blender"
```

Comment:
You changed one to uppercase and the other to lowercase?

Reply:
Must be leftover from tests